examples/computer_vision/deformabledetr_coco_pytorch/startup-hook.sh

```diff
@@ -3,22 +3,26 @@ apt-get install unzip
 
 # Download COCO 2017 annotations
 wget http://images.cocodataset.org/annotations/annotations_trainval2017.zip
-unzip -o annotations_trainval2017.zip 
+unzip -o annotations_trainval2017.zip
 mv annotations/instances_train2017.json /tmp
 mv annotations/instances_val2017.json /tmp
 
-# Clone Deformable-DETR library from source.  
+# Clone Deformable-DETR library from source.
 # Since it is not an installable pacakge, we will have to add this to system path to import functions from it.
 git clone https://github.com/fundamentalvision/Deformable-DETR ddetr
-cd ddetr && git reset --hard 11169a60c33333af00a4849f1808023eba96a931 
+cd ddetr && git reset --hard 11169a60c33333af00a4849f1808023eba96a931
+# Need to fix a bug in the original code that fails to handle torchvision version 0.10 correctly.
+# Deformable DETR has some changes from DETR that need additional handling.
+sed -i 's/float(torchvision\.__version__\[:3\]) < 0.5/int(torchvision\.__version__.split("\.")\[1\]) < 7/g' util/misc.py
```

Comment:
can we upstream a fix to detr?  You should not be parsing versions as floats.

Reply:
i will but the authors of those repos do not seem very responsive so i want to fix our tests first